kitsune/questions/templates/questions/question_details.html

```diff
@@ -118,7 +118,9 @@ <h2 class="summary" itemprop="headline">{{ question.title }}</h2>
         </div>
         <div class="asked-on">
           {{ _('Posted') }}<br>
-          <time itemprop="dateCreated" datetime="{{ question.created }}">{{ datetimeformat(question.created) }}</time>
+					<span itemprop="dateCreated">
+						{{ datetimeformat(question.created) }}
+					</span>
```

Comment:
Though, now that I look closer, why did you change this from `<time>` to `<span>`?